src/Resources/views/profiler.html.twig

```diff
@@ -123,5 +147,14 @@
             $(this).val(parseJson($(this).val()));
         });
 
+        $('.client-selector :checkbox').change(function () {
+            $('.client-selector :checkbox').each(function () {
+                if (!this.checked ) {
+                    $('.row[data-client="' + $(this).val() + '"]').hide();
```

Comment:
I guess `$('.row[data-client="' + $(this).val() + '"]').toggle(this.checked)` could do the trick.
http://api.jquery.com/toggle/#toggle-display